src/arrayMethodSort.js

```diff
@@ -4,8 +4,21 @@
  * Implement method Sort
  */
 function applyCustomSort() {
-  [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+  [].__proto__.sort2 = function (
+    compareFunction = (a, b) => String(a) > String(b),
+  ) {
+    for (let i = 0; i < this.length; i++) {
+      for (let j = 0; j < this.length; j++) {
+        if (compareFunction(this[j], this[j + 1]) > 0) {
+          const sort = this[j];
+
+          this[j] = this[j + 1];
+          this[j + 1] = sort;
+        }
+      }
+    }
```

Comment:
This implementation of bubble sort will always run in O(n^2) time, even if the array is already sorted. To optimize, you can introduce a flag that checks if any swaps have been made in the inner loop; if no swaps are made, the array is already sorted, and you can break out of the loops early.